test/bdd/fixtures/profile/profiles.json

```diff
@@ -172,6 +229,25 @@
       "createDID": true,
       "didDomain": "testnet.orb.local",
       "didServiceAuthToken": "tk1"
+    },
+    {
+      "verifier": {
+        "id": "v_myprofile_jwt",
+        "name": "v_myprofile_jwt",
+        "organizationID": "test_org",
+        "url": "https://test-verifier.com",
+        "active": true,
+        "checks": {
+          "credential": {
+            "format": ["jwt"],
+            "proof": true,
+            "status": false
```

Comment:
We should add some tests with `status = true`.

Reply:
sure, will do

Reply:
@rolsonquadras The VC status check is yet to be implemented, but it's not ready for now. 
I added it as a subpoint in this ticket https://github.com/trustbloc/vcs/issues/757